### ISSUE TITLE
Web and Chromecast support

### ISSUE THREAD

**Issue by utkarshtyagi4** [jhomlala/betterplayer#1154]
Nice player but web support or Chromecast is missing .

**Comment by DieProgrammIDE** [comment]
Would also love to see Web Support especially.
Let me know if i can help!

**Comment by nadveos** [comment]
Yeah....this future maybe most powerful to see work in this package

**Comment by jhomlala** [comment]
Duplicate of https://github.com/jhomlala/betterplayer/issues/64 and https://github.com/jhomlala/betterplayer/issues/1456
